install_config/oab_broker_configuration.adoc

```diff
@@ -375,6 +375,25 @@ registry:
       - ".*-apb$"
 ----
 
+[[oab-config-registry-galaxy]]
+=== Ansible Galaxy Registry
+
+The `galaxy` type allows you to consume APB roles from 
+link:https://galaxy.ansible.com[*Ansible Galaxy*]. You can also optionally
+specify an organization.
+
+[source,yaml]
+----
+registry:
+  - name: galaxy
+    type: galaxy
+    # Optional:
+    # org: ansibleplaybookbundle
+    runner: docker.io/ansibleplaybookbundle/apb-base:latest
+    white_list:
+      - ".*$"
+----
```

Comment:
Others LGTM, one question as below.
In testing, we always using `url`, @dymurray is it a optional ? How about if without `url`?
Such as:
```
    registry:
      - type: galaxy
        name: galaxy
        org: ansibleplaybookbundle
        url: https://galaxy.ansible.com 
        runner: docker.io/ansibleplaybookbundle/apb-base:latest
        white_list: [.*]
```

Reply:
Galaxy adapter using 'url: https://galaxy.ansible.com' as a default value and work well while without 'url' in configuration. So, this PR LGTM. Thx.